tests/integration/test_content_manifest.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import utils
+import pytest
+
+
+def test_invalid_content_manifest_request(test_env, default_request):
+    """
+    Send an invalid content-manifest request to the Cachito API.
+
+    Checks:
+    * Check that response code is 404
+    """
+    client = utils.Client(test_env["api_url"], test_env["api_auth_type"])
+
+    with pytest.raises(Exception) as e:
+        client.fetch_content_manifest(request_id=0)
+    assert "404 Client Error: NOT FOUND" in str(e.value)
```

Comment:
Let's make sure an `HTTPError` is caught. Additionally, you can check the response that caused the issue instead of the exception message.
```suggestion
    with pytest.raises(requests.HTTPError) as e:
        client.fetch_content_manifest(request_id=0)
    assert e.value.response.status_code == 404
```

Ideally, you can also check the error message by Cachito with:
```python
assert e.value.response.json().get("message") == "the expected message goes here"
```

Reply:
Thank you! I am not sure about `e.value.response.json().get("message")`. I used: `str(e.value)`. Is it ok for you?